tools/pylib/boutdata/squashoutput.py

```diff
@@ -99,9 +111,26 @@ def squashoutput(datadir=".", outputname="BOUT.dmp.nc", format="NETCDF4", tind=N
         # move only after the file list is cached
         shutil.move(fullpath,oldfile)
 
+    if docontinue:
+        shutil.move(os.path.join(datadirtmp,outputname),datadir)
+        os.rmdir(datadirtmp)
+
     t_array_index = outputvars.index("t_array")
     outputvars.append(outputvars.pop(t_array_index))
 
+    if progress:
+        sizes_=outputs.sizes()
+        sizes={}
+        total=0
+        for var,size in sizes_.items():
```

Comment:
Please could you put a comment here explaining why you're doing this? Or rename the variables to something more descriptive?